utils/cognitoMigration/tsconfig.json

```diff
@@ -0,0 +1,103 @@
+{
+  "compilerOptions": {
+    /* Visit https://aka.ms/tsconfig to read more about this file */
```

Comment:
Do we want to keep all the commented out lines?

Reply:
I just used an existing template but I clean it up since it won't be necessary